src/main/java/org/sasanlabs/internal/utility/VariantConstants.java

```diff
@@ -0,0 +1,6 @@
+package org.sasanlabs.internal.utility;
+
+public class VariantConstants {
```

Comment:
can we create an Enum ? Also please add Author tag and description of Enum.

Reply:
Unfortunately, it's not possible due to Java limitations. I tried the ```Enum``` approach initially, but the annotation requires providing compile-time constant value, so I had to abandon this idea. The current implementation is similar to what we already have for ```LevelConstants```.

Reply:
Enums are allowed in annotations and they are static constants only. Look at following line https://github.com/SasanLabs/VulnerableApp/blob/ed98eb0923893ea822761cca3e7d0969b0eb340d/src/main/java/org/sasanlabs/internal/utility/annotations/VulnerableAppRequestMapping.java#L48 for more information.

Reply:
I see your point. I tried with different Enum (containing String), but now the simplified approach works. Resolved in 7eca817